web/atlas.json

```diff
@@ -5818,9 +5817,9 @@
-{"id": "txfalv", "submitted_by": "SuperGameController", "name": "Skysail Insignia", "description": "A winged white sword.\n\nIt represents the Kingdom of Skysail in the realm of Inita, one of the many realms in the Countless Worlds.\n\nThe insignia of the winged sword mirrors the great military power of the Skysail Kingdom.\nNot only do they excel in military power, Skysail is the main trading hub within the realms, they control most of the trades between worlds and are responsible for exports of Nillium, a magical ore used in the hull of flying ships.\n\nThe game's discord can be found here > [discord.gg/CW](https://discord.gg/CW) <", "website": "https://www.roblox.com/games/6900861054/Countless-Worlds", "subreddit": "/r/RobloxCountlessWorlds", "center": [297.5, 1508.5], "path": [[293.5, 1515.5], [300.5, 1515.5], [300.5, 1500.5], [293.5, 1500.5], [293.5, 1515.5]]},
-{"id": "txfah1", "submitted_by": "Wellliam", "name": "Emotiguy (Picardia)", "description": "This emotiguy, more famously known as Picardia, had been rebuilt during Xqc's meteor.\nOn day 3, r/OnePiece Added a cool little hat.", "website": "https://trubiso.tk", "subreddit": "/r/emotiguy", "center": [1543.5, 152.5], "path": [[1533.5, 159.5], [1532.5, 142.5], [1550.5, 142.5], [1555.5, 161.5], [1543.5, 161.5]]},
-{"id": "txf9s5", "submitted_by": "AnonymousRandPerson", "name": "Hallgrímskirkja", "description": "The largest church in Iceland.", "website": "https://en.wikipedia.org/wiki/Hallgr%C3%ADmskirkja", "subreddit": "/r/Iceland", "center": [514.5, 126.5], "path": [[515.5, 106.5], [513.5, 108.5], [514.5, 109.5], [514.5, 110.5], [512.5, 112.5], [512.5, 113.5], [511.5, 114.5], [511.5, 116.5], [507.5, 124.5], [507.5, 125.5], [506.5, 126.5], [506.5, 127.5], [503.5, 130.5], [502.5, 130.5], [500.5, 132.5], [500.5, 136.5], [525.5, 136.5], [525.5, 128.5], [524.5, 127.5], [524.5, 126.5], [523.5, 125.5], [523.5, 124.5], [519.5, 116.5], [519.5, 114.5], [518.5, 113.5], [518.5, 112.5], [516.5, 110.5], [517.5, 108.5], [515.5, 106.5]]},
+{"id": "txfalv", "submitted_by": "SuperGameController", "name": "Skysail Insignia", "description": "A winged white sword.\n\nIt represents the Kingdom of Skysail in the realm of Inita, one of the many realms in the Countless Worlds.\n\nThe insignia of the winged sword mirrors the great military power of the Skysail Kingdom.\nNot only do they excel in military power, Skysail is the main trading hub within the realms, they control most of the trades between worlds and are responsible for exports of Nillium, a magical ore used in the hull of flying ships.\n\nThe game's discord can be found here > [discord.gg/CW](https://discord.gg/CW) <", "website": "https://www.roblox.com/games/6900861054/Countless-Worlds", "subreddit": "[/r/RobloxCountlessWorlds", "center": [297.5, 1508.5], "path": [[293.5, 1515.5], [300.5, 1515.5], [300.5, 1500.5], [293.5, 1500.5], [293.5, 1515.5]]},

```

Comment:
Markdown format doesn't work here. Write out links in full without Markdown syntax.

Reply:
Gotcha, will do! Sorry about that

Reply:
so it would just be https://discord.gg/CW then?

Reply:
Yes.